kea_bank/account_management_app/views.py

```diff
@@ -1,95 +1,205 @@
 from decimal import Decimal
 from django.shortcuts import render
+from django.http import Http404
 from . models import Account
 from django.contrib.auth.decorators import login_required
+from django.http import HttpResponseRedirect
+from django.urls import reverse
 
+from . models import Customer
 
 @login_required(login_url='login_app:login')
 def index(request):
-    user = request.user
-    accounts = Account.objects.filter(user=user)
-    total_balance = user.customer.total_balance
+    try:
+        user = request.user
+        accounts = Account.objects.filter(user=user, is_saving_account=False)
+        total_balance = user.customer.total_balance
+
+        return render(request, 'account_management_app/index.html', {
+            'accounts': accounts,
+            'customer': user,
+            'total_balance': total_balance
+        })
+    except Customer.DoesNotExist:
```

Comment:
Could this exception handling be extracted (e.g. to some middleware) so we don't have to handle it in every view?